### ISSUE TITLE
Problem: Provide a solution for indexing submission docs that contain advanced mathematical notation in Elasticsearch

### ISSUE THREAD

**Issue by currmie** [archivematica/Issues#1583]
Provide a solution that would enable custom filtration of restricted characters or semantics commonly found in advanced mathematics AND in documents included within the objects or submissionDocumentation subdirectories of a transfer package. Doing so would enhance the indexing of documentation and datasets across multiple disciplines in the sciences and particular areas of social sciences.

Ideally, there would be some research into built-in tokenizers or custom analyzers within the Elasticsearch capabilities to filter certain patterns in advanced mathematical notation. Additionally, enhancements of the existing Archivematica developer documentation to include sample user match queries for text strings or text analyzed fields would be an added bonus.

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] A testing plan for this issue has been implemented and passed (testing plan information should be included in the issue body or comments)
- [ ] Documentation regarding this issue has been written and merged
- [ ] Details about this issue have been added to the release notes

**Comment by cmhosale** [comment]
Commenting as part of the issue-closure-2025 process. This feature request remains a priority for MIT. Please consider it for development in a future release.

**Comment by sromkey** [comment]
@cmhosale We're curious for more information on this use case- we currently do not index the full text of either  submission documentation or objects at all, regardless of character set. Some more context about your exact needs would be helpful.

**Comment by jfcarrano** [comment]
@sromkey I dealt with this issue a few years ago. It has to do with the `metadata.csv` imported in the `metadata` directory. We had a `dc.description.abstract` that contained mathematical symbols that was causing indexing to fail and the ingest to fail as a result. We confirmed that we were importing valid UTF-8, which was transformed by Archivematica into valid XML. The issue was with elasticsearch dealing with the symbols. Specifically it was a multiplication symbol after a parentheses in this element `log(1/𝜀)· 𝑁¹⁻ᶿ⁽¹⸍ ˡᵒᵍ⁽¹⸍𝜀⁾` we were able to solve the issue by adding a space in between.

Our developers felt this was a bug with Archivematica and not the data we produce, which was in response to the suggestion that we find a way to change our workflows to strip out this data. They felt that if Archivematica has internal parsing rules it needs to apply in order to index content, it should handle the necessary data changes as part of that process. Additionally, we use this same content in OpenSearch (which is for all intents and purposes ElasticSearch with a different name) and don’t run into these issues. You can see the metadata record successfully loaded into our local OpenSearch here: https://search.libraries.mit.edu/record/dspace:1721.1-139095

**Comment by sromkey** [comment]
@jfcarrano sorry for the delay following back up- so the problem in Archivematica could be said to be more about the indexing failure (hence preventing other metadata from being searched) than about the ability to index these characters specifically? To put it more plainly, would I be correct to assume there should be no expectation that users can search Archival Storage for the mathematical notation itself?

**Comment by jfcarrano** [comment]
@sromkey correct, highly unlikely someone would be searching for those symbols in Archival Storage
